native/common/include/jp_booleantype.h

```diff
@@ -0,0 +1,68 @@
+/*****************************************************************************
```

Comment:
Each of the primitive types were split into separate files to improve management.  These "autogenerated" files were so heavily edited that they were drifting far apart.  As separate files they are much more manageable.